tokio/src/runtime/scheduler/current_thread.rs

```diff
@@ -67,7 +67,7 @@ struct Core {
 /// Scheduler state shared between threads.
 struct Shared {
     /// Remote run queue. None if the `Runtime` has been dropped.
```

Comment:
I think the comment that this is `None` if the runtime has been dropped needs to be removed, as this is no longer an `Option`.

Reply:
Done